configurations/performance/cassandra_stress_gradual_load_steps_enterprise.yaml

```
@@ -0,0 +1,4 @@
+# Define load ops for steps
+perf_max_ops: {"read": 794000, "write": 405000, "mixed": 530000}
+perf_threads: {"read": 620, "write": 700, "mixed": 1900}
```

for write I asked to use "400".
```suggestion
perf_threads: {"read": 620, "write": 400, "mixed": 1900}
```